app/models/concerns/host_searching.rb

```diff
@@ -5,7 +5,7 @@ module HostSearching
   extend ActiveSupport::Concern
 
   included do
-    scoped_search on: %i[display_name os_major_version os_minor_version]
+    scoped_search on: %i[id display_name]
     scoped_search on: :display_name, rename: :name
     scoped_search on: :compliant, ext_method: 'filter_by_compliance',
                   only_explicit: true
```

Comment:
`only_explicit` is here for fields which we don't want to search through without their keyword

Reply:
Is your suggestion to move those os lookups to `only_explicit`?

Reply:
Pretty much everything should be `only_explicit` IMO. In this case, we have to continue supporting `display_name`/`name` search implicitly.